modules/upgrade/pages/proxy-upgrade.adoc

```diff
@@ -0,0 +1,40 @@
+[[update.suse.manager.proxy]]
+= Update {productname} Proxy
+
+{productname} Proxies are managed as clients by {productname}.
+Thus patches and package updates combined as maintenance updates (MU) are to be installed with {productname} as it is done with other {productname} clients.
+When the update is installed, you must check whether a restart of the proxy service is required.
+
+// a remote command (or a salt state) can perform the restart.
+// Creating an Action Chain can help with this task.
+// With an action chain you can update proxies on batches, or all at the same time:
+// These are the basic steps to run:
+////
+1. Run a salt command: `spacewalk-proxy stop`
+2. Update the packages
+3. Run a salt command: `spacewalk-proxy start`
+////
+
+
+
+== Before You Begin
```

Comment:
```suggestion
== Prepare to Upgrade
```

We really need to standardise this heading across the suite. We use lots of different wording for pre-reqs.